ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
but this is 1.16 too ...

Reply:
yes relic from the previous setup will update

Reply:
@nak3 do we care about it? I just updated to 1.17 any concerns?

Reply:
seems no